assets/src/edit-story/components/movable/index.js

```diff
@@ -97,11 +98,15 @@ function Movable( {
 				setPropertiesOnSelectedElements( newProps );
 				resetMoveable( target );
 			} }
-			onResizeStart={ ( { setOrigin, dragStart } ) => {
+			onResizeStart={ ( { setOrigin, dragStart, direction } ) => {
 				setOrigin( [ '%', '%' ] );
 				if ( dragStart ) {
 					dragStart.set( frame.translate );
 				}
+				const newKeepRatioMode = direction[ 0 ] !== 0 && direction[ 1 ] !== 0;
```

Comment:
Could we add a comment here explaining what this means -- `direction[ 0 ]` and `direction[ 1 ]`?

Reply:
Done